curriculum/challenges/english/25-front-end-development/workshop-ferris-wheel/6140d0069049f5139d78da40.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 6140d0069049f5139d78da40
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+The `transform-origin` property is used to set the point around which a CSS transformation is applied. For example, when performing a `rotate` (which you will do later in this project), the `transform-origin` determines around which point the element is rotated.
+
+Give the `.line` selector a `transform-origin` property of `0% 0%`. This will offset the origin point by `0%` from the left and `0%` from the top, setting it to the top left corner of the element.
```

Comment:
```suggestion
Give the `.line` selector a `transform-origin` property of `0% 0%`. This will offset the origin point at `0%` from the left and `0%` from the top, setting it to the top left corner of the element.
```